packages/table/src/cell/editableCell.tsx

```diff
@@ -49,30 +49,42 @@ export interface IEditableCellProps extends ICellProps {
 }
 
 export interface IEditableCellState {
-    isEditing: boolean;
+    isEditing?: boolean;
+    savedValue?: string;
+    dirtyValue?: string;
 }
 
 export class EditableCell extends React.Component<IEditableCellProps, IEditableCellState> {
-    public state = {
-        isEditing: false,
-    };
+    public state: IEditableCellState;
```

Comment:
no need to declare member, only need to do this if you're also defining it. but since you're defining in constructor, please remove this.